### ISSUE TITLE
We should have a diagnostic for missing whitespace around C# 6 property syntax

### ISSUE THREAD

**Issue by AArnott** [DotNetAnalyzers/StyleCopAnalyzers#741]
No diagnostic was produced for not having exactly one space between the `}` and `=` in the snippet below.

```csharp
public static PropertySerializer SimpleTypes { get; }= InitializeSimpleTypes();
```

I suspect there is an existing rule that can be enhanced to handle this C# 6 case rather than creating a new rule, although I'm not sure which rule it would be

**Comment by pdelvo** [comment]
SA1003 is used to report spacing violations around '=' so this would be the rule that should be extended

**Comment by sharwell** [comment]
SA1003 already does this according to the definition. You aren't seeing it in Alpha 5 because #665.

**Comment by AArnott** [comment]
How can #665 explain why I'm not seeing it in alpha005 when that only contains changes to tests? 
Also I don't see in those tests an instance of my example above. All those are dealing with operators that appear within a method. This appears as a property declaration.

**Comment by vweijsters** [comment]
I've added some testcases for auto property initializers to #665.
The SA1003 analyzer is not reporting diagnostics for this case, as already witnessed.

@sharwell Did you close this issue because its basically a duplicate of #665?

**Comment by sharwell** [comment]
@AArnott SA1003 is disabled in the release build because it has no tests, hence having it not report your issue. I closed this as a duplicate of the open request to implement SA1003.

**Comment by sharwell** [comment]
@vweijsters Thanks for adding those tests. Yes, there are known bugs in the implementation of SA1003 which will need to be addressed before we enable it.
